src/AllureReporter.js

```diff
@@ -29,6 +29,7 @@ function AllureReporter(baseReporterDecorator, config) {
                     return Math.max(stop, result.stop);
                 }, Number.NEGATIVE_INFINITY);
             this.allure.endSuite(stopTime);
+            delete this.suites[suite];
```

Comment:
why you decide to skip suites cleanup?

Reply:
This deletion resolve issue #12 .
When I used watch mode - karma crashed. 
Because suite already ended, but `this.suite` not cleared. And next start on this line https://github.com/allure-framework/allure-karma/blob/2e86a37eff25cb50bf9c922325e23acac1bb5edd/src/AllureReporter.js#L78 allure-karma thinks that the suite is not empty and does not create a new one.